server/models/postgis/project.py

```diff
@@ -135,6 +139,30 @@ def set_default_changeset_comment(self):
         self.changeset_comment = f'{default_comment}-{self.id} {self.changeset_comment}' if self.changeset_comment is not None else f'{default_comment}-{self.id}'
         self.save()
 
+    def set_country_info(self):
+        """ Sets the default country based on centroid"""
+        query = """SELECT ST_Y(ST_Transform(ST_Centroid(centroid), 4326)) || ', '
```

Comment:
@ramyaragupathy I think we can use sqlalchemy here. Something like a static method similar to https://github.com/hotosm/tasking-manager/blob/develop/server/models/postgis/project.py#L632 :
```
lat, lng = db.session.query(func.ST_Y(Project.centroid), func.ST_X(Project.centroid)).filter(Project.id==project_id).one()
```

Reply:
Thanks for the flag @JorgeMartinezG. Replaced the centroid query and also the country fetch query